funcsigs/__init__.py

```diff
@@ -20,8 +20,8 @@
 __all__ = ['BoundArguments', 'Parameter', 'Signature', 'signature']
 
 
-_WrapperDescriptor = type(type.__call__)
-_MethodWrapper = type(all.__call__)
+_WrapperDescriptor = type(bytearray.__add__)
+_MethodWrapper = type(u'str'.__add__)
```

Comment:
Emm... This is because in Pyston, the type of `'str'.__add__` still is `instancemethod`.
```
Pyston v0.6.0 (rev 8dd1298946b5dbe2b28f75a7a6d2de6486765438), targeting Python 2.7.7
>> type('str'.__add__)
<type 'instancemethod'>
>> type(u'str'.__add__)
<type 'method-wrapper'>
```